be/test/exprs/lambda_map_expr_test.cpp

```diff
@@ -251,7 +251,7 @@ TEST_F(MapApplyExprTest, test_map_int_int) {
         ColumnPtr result = map_apply_expr->evaluate(&exprContext, &cur_chunk);
 
         EXPECT_TRUE(result->is_nullable());
-        EXPECT_STREQ(result->debug_string().c_str(), "[{0:67}, {0:89}, {0:NULL}, {}, NULL]");
+        EXPECT_STREQ(result->debug_string().c_str(), "[{0:45}, {0:78}, {0:NULL}, {}, NULL]");
```

Comment:
why change these tests? the function should keep the last win

Reply:
There is no order in the map definition. So I don't think keeping the last win is necessary.

Reply:
trino keep the last win

Reply:
Ok, I will change this to last win

Reply:
Done